### PR TITLE
ci(k8s-eks): add CI job configuration for the EKS multiDC setups

### PR DESCRIPTION
**Pull request by vponomaryov**
Add 2 CI jobs:
- 3h  using the `i4i.2xlarge` instance type for DB-specific pods
- 12h using the `i4i.4xlarge` instance type for DB-specific pods

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by vponomaryov** [comment]
12h job example (same as in https://github.com/scylladb/scylla-cluster-tests/pull/6820#issuecomment-1816220502):

Tested here: [Argus](https://argus.scylladb.com/test/e9f07277-fc10-4a28-9f2a-cec9e662b5e9/runs?additionalRuns[]=842b591e-04a4-493f-86a0-b37dca32ad69), [CI](https://jenkins.scylladb.com/job/scylla-staging/job/valerii/job/vp-longevity-scylla-operator-multidc-12h-eks/9/consoleFull)

Nemesis list from the passed job:
Screenshot from 2023-11-17 13-54-48
Screenshot from 2023-11-17 13-54-28

Monitoring:
Screenshot from 2023-11-17 13-57-18

**Comment by vponomaryov** [comment]
3h job example:
[Argus](https://argus.scylladb.com/test/d47b5585-746c-493f-8b3c-d5ff562450cd/runs?additionalRuns[]=e711ee88-bb90-419a-9b7c-1818bf521fb0), [CI](https://jenkins.scylladb.com/job/scylla-staging/job/valerii/job/vp-longevity-scylla-operator-multidc-3h-eks/51/)
Screenshot from 2023-11-17 14-04-52

**Review by fruch — APPROVED** [review]
LGTM

File: test-cases/scylla-operator/longevity-scylla-operator-multidc-12h.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "cassandra-stress write cl=QUORUM n=10240000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=3000 -col 'n=FIXED(2) size=FIXED(512)' -pop seq=10240001..20480000 -log interval=5",
+]
+stress_cmd: [
+  "cassandra-stress mixed cl=LOCAL_QUORUM duration=720m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=50 -pop 'dist=uniform(1..15360000)' -col 'n=FIXED(2) size=FIXED(512)' -log interval=5",
+  "cassandra-stress mixed cl=LOCAL_QUORUM duration=720m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=50 -pop 'dist=uniform(5120001..20480000)' -col 'n=FIXED(2) size=FIXED(512)' -log interval=5",
+]
+round_robin: true
+
+n_db_nodes: 4
```
Comment (soyacz) [inline test-cases/scylla-operator/longevity-scylla-operator-multidc-12h.yaml:13]: shouldn't we specify number of nodes per dc in a list?

Comment (vponomaryov) [reply test-cases/scylla-operator/longevity-scylla-operator-multidc-12h.yaml:13]: > shouldn't we specify number of nodes per dc in a list?

It is not supported yet.

Comment (fruch) [reply test-cases/scylla-operator/longevity-scylla-operator-multidc-12h.yaml:13]: @vponomaryov let's open a task for it, and it's not urgent for us now

Comment (vponomaryov) [reply test-cases/scylla-operator/longevity-scylla-operator-multidc-12h.yaml:13]: @fruch https://github.com/scylladb/qa-tasks/issues/1563

**Pull request merged**